src/cpp/database/entities.cpp

```diff
@@ -13,12 +13,63 @@
  * limitations under the License.
  */
 
+#include <set>
+
 #include "entities.hpp"
 
 namespace eprosima {
 namespace statistics_backend {
 namespace database {
 
+bool is_metatraffic_topic(
+        std::string topic_name)
+{
+    bool is_metatraffic = false;
+    std::set<std::string> metatraffic_topics = {
+        "___EPROSIMA___METATRAFFIC___DOMAIN_0___TOPIC",
```

Comment:
This topic name is probably an error and it will change in the future
```suggestion
        "___EPROSIMA___METATRAFFIC___",
```